nginx/Dockerfile

```diff
@@ -12,17 +12,21 @@ RUN chmod 755 /docker-entrypoint.d/ssl-install.sh
 COPY templates /etc/nginx/templates
 COPY nginx.conf /etc/nginx/nginx.conf
 COPY mime.types /etc/nginx/mime.types
+COPY conf.d/ /etc/nginx/conf.d/
+COPY nginx_error_pages/ /usr/share/nginx/nginx_error_pages/
 
 # test build
-FROM ghcr.io/ffurrer2/bats:1.6.0 as test_nginx
-RUN apk --no-cache add curl openssl
+FROM alpine:3 as test_nginx
+RUN apk --no-cache add bats curl openssl
 RUN mkdir -p /app/bash-shellmock
 RUN curl -LJO https://raw.githubusercontent.com/capitalone/bash_shell_mock/main/bin/shellmock
 RUN cp shellmock /app/bash-shellmock/shellmock
 RUN chmod +x /app/bash-shellmock/shellmock
-COPY ssl-install.sh /app/ssl-install.sh
+COPY tests /app/tests
+COPY ssl-install.sh /app/tests/ssl-install.sh
+RUN bats /app/tests/tests.bats
 ENV PATH="${PATH}:/app/"
-ENTRYPOINT [ "/bin/bash", "/usr/local/bin/bats", "/app/tests.bats" ]
+ENTRYPOINT [ "/bin/bash", "/usr/bin/bats", "/app/tests/tests.bats" ]
```

Comment:
so - I have to confess - I know what this code does, but I couldn't quite figure where it's use, or even if it's used.  I suspect it's my lack of understanding of docker and our test build environment!

I assumed if I ran `make start` I could then run `docker exec -it cht-nginx bats /app/tests/tests.bats` to see the tests run inside the container.  However, I get:

```
docker exec -it cht-nginx bats /app/tests/tests.bats
/usr/libexec/bats-core/bats: line 87: cd: /app/tests: No such file or directory
```

suggesting that line 25 did work?!  Maybe this is to be expected?

Reply:
The entrypoint can probably be removed, it was used in the old compose file to only run tests rather than starting the service. The reason you can't find `app/tests` is it doesn't exist is that the build target `test_nginx` is not included in the final container

Reply:
Entrypoint and path for test image removed